.env.example

```diff
@@ -155,3 +155,8 @@ OPENDAL_AZFILE_ROOT=/tmp/opendal/
 OPENDAL_AZFILE_ACCOUNT_NAME=<account_name>
 OPENDAL_AZFILE_ACCOUNT_KEY=<account_key>
 OPENDAL_AZFILE_SHARE_NAME=<shared_name>
+# gridfs
+OPENDAL_GRIDFS_CONNECTION_STRING=mongodb://localhost:27017
+OPENDAL_GRIDFS_DATABASE=opendal
```

Comment:
How about using `<database>` to make it more clear that it's a value need to fill?